modules/flight_interface/flight_interface.py

```diff
@@ -35,6 +38,13 @@ def create(cls, address: str, timeout_home: float) -> "tuple[bool, FlightInterfa
         # Get Pylance to stop complaining
         assert home_location is not None
 
+        result, flight_interface_logger = logger.Logger.create("flight_interface")
+
+        if result:
+            frame = inspect.currentframe()
+            flight_interface_logger.info("flight interface logger initialized", frame)
+            flight_interface_logger.info(home_location, frame)
```

Comment:
I think this should be 1 message, since it's 1 action (and 1 frame)

Reply:
Fixed.